### ISSUE TITLE
Error: WHERE parameter "thread_id" has invalid "undefined" value

### ISSUE THREAD

**Issue by Rotorsoft** [hicommonwealth/commonwealth#9581]
View details in Rollbar: [https://app.rollbar.com/a/commonwealth/fix/item/Commonwealth/9548](https://app.rollbar.com/a/commonwealth/fix/item/Commonwealth/9548)

```

Error: WHERE parameter "thread_id" has invalid "undefined" value
  File "/prod/commonwealth/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/src/dialects/abstract/query-generator.js", line 2445, in PostgresQueryGenerator.whereItemQuery
  File "/prod/commonwealth/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/src/dialects/abstract/query-generator.js", line 2434, in <anonymous>
  File "<anonymous>", line unknown, in Array.forEach
  File "/prod/commonwealth/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/src/dialects/abstract/query-generator.js", line 2432, in PostgresQueryGenerator.whereItemsQuery
  File "/prod/commonwealth/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/src/dialects/abstract/query-generator.js", line 2860, in PostgresQueryGenerator.getWhereConditions
  File "/prod/commonwealth/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/src/dialects/abstract/query-generator.js", line 1460, in PostgresQueryGenerator.selectQuery
  File "/prod/commonwealth/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/src/dialects/abstract/query-interface.js", line 1002, in PostgresQueryInterface.select
  File "/prod/commonwealth/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/src/model.js", line 1824, in Function.findAll
  File "node:internal/process/task_queues", line 95, in process.processTicksAndRejections
  File "/prod/commonwealth/server/controllers/server_threads_methods/get_thread_polls.ts", line 13, in async ServerThreadsController.__getThreadPolls
      const polls = await this.models.Poll.findAll({
```

**Comment by Rotorsoft** [comment]
@Israellund this has been showing in rollup, seems to be related to the client sending an invalid threadid

I tried to reproduce it on master without luck

**Comment by Israellund** [comment]
I've tried replicating this on both local and prod, but have had no luck in being able to pinpoint what actions on the site may be causing this. It seems to be somewhere around `ViewThreadPage.tsx` which calls `useGetThreadPollsQuery`, yet after extensive clicking around threads I was unable to induce the error in Rollbar.. In Rollbar shows it shows it coming from `get_thread_polls.ts` (see screenshot below). I have followed the `useGetThreadPollsQuery` call from the front to the back, logging `threadId` as I go, as well as checking all files for any errors (linting, logic, etc), but still could not find anything that might be causing this error. 
If anyone else wants to take a stab at this, @timolegros @kurtisassad @mzparacha let me know, otherwise I will close this ticket as it doesn't seem to be affecting the site in any other way other than showing an error in Rollbar. 
![Screenshot 2024-10-18 at 1 05 26 PM](https://github.com/user-attachments/assets/3b145d58-d423-42d6-b30a-2d901b466e5f)
